### PR TITLE
Add notes for CG 2023-11-07

### PR DESCRIPTION
**Pull request by dschuff**

### CODE REVIEW

**Comment by dschuff** [comment]
@dtig it doesn't look like there are any notes from the frozen proposal part in the doc; did that get postponed?

File: main/2023/CG-11-07.md [hunk 1]
```diff
@@ -39,4 +39,122 @@ Installation is required, see the calendar invite.
...
+ - Slava Kuzmich
+ - Talg
+ - Yuri Iozzelli
+ - Yury Delendik
+ - Zalim Bashorov
+
+
+### Proposals and discussions
+
+### Memory control: Proposal update and Implementation report (Deepti Gandluri) [slides]()
```
Comment (dschuff) [inline main/2023/CG-11-07.md:91]: @dtig is there a link?

Comment (dtig) [reply main/2023/CG-11-07.md]: Added link directly to the notes

**Comment by dtig** [comment]
Thanks for adding notes!

> @dtig it doesn't look like there are any notes from the frozen proposal part in the doc; did that get postponed?

Yeah, It got postponed to the 12/5 meeting.

**Pull request merged**